docs/CHANGELOG.asciidoc

```diff
@@ -32,6 +33,376 @@ This section summarizes the changes in each release.
+Logs::
+* Log rate setup index validation {pull}50008[#50008]
+* Adds categories table to the categorization tab {pull}53004[#53004]
+* Disables ML job setup form while setup is pending {pull}54705[#54705]
+Machine Learning::
+* Updates Anomaly Detection job wizard button styles, page panel and titles {pull}53340[#53340]
+* Improves processing of groups in data recognizer wizard {pull}49310[#49310]
+* Indicates missing required privileges for import in File Data Viz {pull}50147[#50147]
+* Stats bar for data frame analytics {pull}49464[#49464]
+* Adds the job message tab to data frame analytics {pull}50468[#50468]
+* Reactive time-range selection in SMV {pull}51008[#51008]
+* Adds Custom URLs and Calendars to the Job wizards {pull}51281[#51281]
+* Performs cardinality check on enabling the model plot {pull}51915[#51915]
+* Fetches the latest job messages and enable sorting by time {pull}52388[#52388]
+* Persists the time range brush when expanded to full width {pull}54020[#54020]
+* Supports search for partitions on Single Metric Viewer {pull}53879[#53879]
+* APM modules configs for RUM Javascript and NodeJS {pull}53792[#53792]
+* MML calculator enhancements for multi-metric job wizard  {pull}54573[#54573]
+* Data Visualizer redesign {pull}54358[#54358]
+* Formatting for additional timing and model size stats {pull}55062[#55062]
+* Adds ML node warning to overview and analytics pages {pull}50766[#50766]
+* Enables lat_long detector function in advanced wizard {pull}50787[#50787]
+* Categorization wizard {pull}53009[#53009]
+* Regression results view {pull}49667[#49667]
+* Auto-populate model_memory_limit {pull}50714[#50714]
+* Reruns evaluate endpoint for search bar queries {pull}50991[#50991]
+* Adds search bar {pull}51235[#51235]
+* Creates classification jobs via the UI {pull}51619[#51619]
+* Adds description field to job creation and display in jobs list {pull}52217[#52217]
+* Creates classification jobs results view {pull}52584[#52584]
+* Adds 'excludes' input field to form {pull}53856[#53856]
+* Adds link to docs {pull}54189[#54189]
+* Uses field caps api to set column type {pull}54543[#54543]
+* Truncates text in Overview page 'Latest timestamp' column {pull}50004[#50004]
+* Color Range Legend component {pull}52794[#52794]
+Management::
+* Support scheme field when creating a Threshold alert with a Webhook action {pull}53757[#53757]
+* Adds Mappings Editor to Index Template Wizard {pull}47562[#47562]
+* SearchProfiler to NP {pull}48795[#48795]
+* Upgrade Assistant to New Platform Ready {pull}50163[#50163]
+* License Management to New Platform {pull}51886[#51886]
+* New Platform (NP) Migration {pull}50908[#50908]
+* Console with better SQL support {pull}51446[#51446]
+* Allows User to Cleanup Repository from UI  {pull}53047[#53047]
+* Updates advanced settings text for usage data {pull}52657[#52657]
+Maps::
+* Data load event handlers {pull}49373[#49373]
```

Comment:
I'd remove this line. It does not affect end-users, and is for internal use only.